contracts/PathsDB.sol

```diff
@@ -26,4 +26,8 @@ contract PathsDB {
 
       return checkpoints;
     }
+
+    function getPathLength() constant returns (uint) {
```

Comment:
@DavidKnott Trying to wrap my head around this - was this a refactor, separating this into its own function, or does extracting this function serve a different purpose?